examples/panantarctic_regional_simulation.jl

```diff
@@ -0,0 +1,233 @@
+using ClimaOcean
+using ClimaOcean.ECCO
+using Oceananigans
+using Oceananigans.Units
+using Oceananigans.Grids: φnode
+using CairoMakie
+using CFTime
+using Dates
+using Printf
+using CUDA
+
+arch = GPU()
+Nx = 1440
+Ny = 400
+Nz = 40
+
+depth = 6000meters
+z = ExponentialCoordinate(Nz, -depth, 0)
+
+grid = LatitudeLongitudeGrid(arch;
+                             size = (Nx, Ny, Nz),
+                             halo = (7, 7, 7),
+                             z = z,
+                             latitude  = (-80, -20),
+                             longitude = (0, 360))
+
+bottom_height = regrid_bathymetry(grid;
+                                  minimum_depth = 10meters,
+                                  interpolation_passes = 7,
+                                  major_basins = 1)
+ 
+grid = ImmersedBoundaryGrid(grid, GridFittedBottom(bottom_height); active_cells_map=true)
+
+h = grid.immersed_boundary.bottom_height
+
+fig, ax, hm = heatmap(h, colormap=:deep, colorrange=(-depth, 0))
+Colorbar(fig[0, 1], hm, label="Bottom height (m)", vertical=false)
+save("acc_bathymetry.png", fig)
+
+start_date = DateTime(1993, 1, 1)
+end_date   = DateTime(1993, 12, 1) 
+
+#
+# Restoring force 
+#               φS                   φN             -20
+# -------------- | ------------------ | ------------ |
+# no restoring   0    linear mask     1   mask = 1   1
+#
+
+const φN₁ = -23
+const φN₂ = -25
+const φS₁ = -78
+const φS₂ = -75
+
+@inline northern_mask(φ)    = min(max((φ - φN₂) / (φN₁ - φN₂), zero(φ)), one(φ))
+@inline southern_mask(φ, z) = ifelse(z > -20, 
+                                     min(max((φ - φS₂) / (φS₁ - φS₂), zero(φ)), one(φ)), 
+                                     zero(φ))
+
+@inline function tracer_mask(λ, φ, z, t)
+     n = northern_mask(φ)
+     s = southern_mask(φ, z)
+     return max(s, n)
+end
+
+@inline function u_restoring(i, j, k, grid, clock, fields, p)
+     φ = φnode(i, j, k, grid, Face(), Center(), Center())
+     return - p.rate * fields.u[i, j, k] * northern_mask(φ)
+end
+
+@inline function v_restoring(i, j, k, grid, clock, fields, p)
+     φ = φnode(i, j, k, grid, Center(), Face(), Center())
+     return - p.rate * fields.v[i, j, k] * northern_mask(φ)
+end
+
+T_meta = Metadata(:temperature; start_date, end_date, dataset = ECCO4Monthly())
+S_meta = Metadata(:salinity;    start_date, end_date, dataset = ECCO4Monthly())
```

Comment:
I'm debating whether to use ECCO2 here for two reasons:

- it's 1/4 resolution so much more vigorous eddy field
- to differentiate this example from the near global
- perhaps we are able to also initialise u and v?

Reply:
But we can also leave this change for a later PR!

Reply:
For sponge layers that are pure relaxation, we typically want to relax to climatology rather than an eddying solution. If we have proper open boundaries (which is not supported yet), then it will make sense to pair the open boundary condition + sponge layer with the resolution of the simulation. (But this comment could only be referring to initial conditions?)